jobs/build-arch.Jenkinsfile

```diff
@@ -225,6 +228,10 @@ lock(resource: "build-${params.STREAM}-${params.ARCH}") {
                 cosa remote-session sync \${dir}/ :\${dir}/
             fi
 
+            # sync over any certificates we have over
+            cosa remote-session sync /etc/pki/ca-trust/anchors/ :/etc/pki/ca-trust/source/anchors
+            cosa shell -- /usr/lib/coreos-assembler/update-ca-trust-unpriv
+
```

Comment:
I'm wondering if rather than doing this here we actually do it inside of `pipeutils.addOptionalRootCA()` and just check for `$COREOS_ASSEMBLER_REMOTE_SESSION` in the environment and take the approprate action based on that.

Reply:
OK, updated this to do that!